src/App.tsx

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
-
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import { PostList } from './components/PostList/PostList';
```

Comment:
You use transit files index.tsx, but you don't make links to them, but to the file itself with the component. It is better to use extension ts for transit files and file types but you use tsx.

Reply:
`+`